Source/DataSources/KmlDataSource.js

```diff
@@ -1846,6 +2384,210 @@ define([
         });
     };
 
+    function getNetworkLinkUpdateCallback(dataSource, networkLink, newEntityCollection, networkLinks) {
+        return function(rootElement) {
+            var remove = false;
+            var networkLinkControl = queryFirstNode(rootElement, 'NetworkLinkControl', namespaces.kml);
+            var hasNetworkLinkControl = defined(networkLinkControl);
+
+            var minRefreshPeriod = 0;
+            if (hasNetworkLinkControl) {
+                if (defined(queryFirstNode(networkLinkControl, 'Update', namespaces.kml))) {
+                    console.log('KML - NetworkLinkControl updates aren\'t supported.');
+                    networkLink.updating = false;
+                    networkLinks.splice(networkLinks.indexOf(networkLink), 1);
+                    return;
+                }
+                networkLink.cookie = defaultValue(queryStringValue(networkLinkControl, 'cookie', namespaces.kml), '');
+                minRefreshPeriod = defaultValue(queryNumericValue(networkLinkControl, 'minRefreshPeriod', namespaces.kml), 0);
+            }
+
+            var now = JulianDate.now();
```

Comment:
I assume all you really need here is a timestamp and not a full blown `JulianDate`.  I would use `getTimestamp` instead (it's in Core) everywhere you are using `JulianDate.now()`.  It will be faster, have less GC pressure and returns milliseconds so you can do basic math instead of calling JD functions.